CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v0.6.5 Release
+
+### Features:
+
+* Added loadBalancer field in spec to Load Balancer Support
```

Comment:
Added loadbalancer field in spec to <provide> Load balancer Support.
OR Added loadbalancer field in spec <for supporting> Load balancer configuration

Reply:
Added loadBalancer field in spec for Load Balancer Support